app/v1/models/des/CreateLossClaimsResponse.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v1.models.des
+
+import play.api.libs.json.{Json, Reads, Writes, __}
+
+case class CreateLossClaimsResponse(id: String)
```

Comment:
Singular: CreateLossClaimResponse

Reply:
A loss can have multiple claims, same as per the URL

Reply:
Yes. But I understood this request to be for a single claim.